corehq/util/workbook_json/const.py

```diff
@@ -1 +1 @@
-MAX_WORKBOOK_ROWS = 1_000_000
+MAX_WORKBOOK_ROWS = 1_048_576  # max number of rows in Excel
```

Comment:
Is this change necessary? If it's the limit for Excel then there's no way anyone could submit a file with more rows with this, right?

Or were you trying to put a limiting mechanism in place with the initial limit set as high as possible? Either way, I think it's fine to keep our limit at 1M rows.

Reply:
> Or were you trying to put a limiting mechanism in place with the initial limit set as high as possible? 

Yeah exactly. It felt a bit heavy handed to implement a generic limit that is lower than what it has effectively been without more research/comms, so agree this is effectively making this generic limit moot. If there are workflows where we can support the full 1,048,576 row limit that excel provides, it seems like a better UX to allow that than to arbitrarily limit it to 1 million rows.

This is all under the assumption that we are actively working on implementing specific row limits for features where we have experienced issues in the past, so the problematic areas will be addressed in the near future.